File: App/src/app/_core/_model/account.group.ts
import { Account } from "./account";
import { Period } from "./period";

export interface AccountGroup {
  id: number;
  name: string;
  createdBy: number;
  modifiedBy: number;
  createdTime: string;
  position: number;
  modifiedTime: string | null;
  accounts: Account[];
  periods: Period[];
}

File: ScoreKPI/Models/AccountGroup.cs
using ScoreKPI.Models.Interface;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ScoreKPI.Models
{
    [Table("AccountGroups")]
    public class AccountGroup: IDateTracking
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public int Position { get; set; }
        public int CreatedBy { get; set; }
        public int ModifiedBy { get; set; }
        public DateTime CreatedTime { get ; set ; }
        public DateTime? ModifiedTime { get ; set ; }
        public virtual ICollection<Account> Accounts { get; set; }
        public virtual ICollection<Period> Periods { get; set; }
    }
}
